src/app/features/courses/courses.component.html

```diff
@@ -0,0 +1,7 @@
+<app-course *ngFor='let courseItem of courseMockList; trackBy:trackById'>
```

Comment:
app course should has Input = course and should be responsible for displaying all the data, rather using content projection